packages/node_modules/@ciscospark/widget-roster/src/reducer.js

```diff
@@ -1,16 +1,22 @@
 import {fromJS} from 'immutable';
 import {combineReducers} from 'redux';
 
+import avatar from '@ciscospark/redux-module-avatar';
 import conversation from '@ciscospark/redux-module-conversation';
 import user from '@ciscospark/redux-module-user';
+import search from '@ciscospark/redux-module-search';
 import spark from '@ciscospark/react-redux-spark';
 
 import {
-  UPDATE_WIDGET_STATE
+  UPDATE_WIDGET_STATE,
+  VIEW_MAIN
 } from './actions';
 
 export const initialState = fromJS({
-  widgetState: {}
+  widgetState: {
```

Comment:
Does this need to be down a level? I think `currentView` and `searchTerm` can be at the top level since nothing else is in this reducer.